src/natcap/invest/coastal_vulnerability.py

```diff
@@ -1676,7 +1676,7 @@ def zero_negative_values(depth_array, nodata):
     """Convert negative values to zero for relief."""
     result_array = numpy.empty_like(depth_array)
     if nodata is not None:
-        valid_mask = ~numpy.isclose(depth_array, nodata)
+        valid_mask = depth_array != nodata

```

Comment:
@emlys this change seems weirdly out of scope of this PR. Plus I have a memory of changing this _to_ `isclose` just prior to the 3.9 release. So maybe this branch still needs to merge in the upstream `main`?

Reply:
Weird, that definitely shouldn't be in there. And it's a float array so `isclose` is correct. Thanks for catching it!